source/_static/clipboard-driver.js

```diff
@@ -0,0 +1,44 @@
+
+//Grab all elements with highlight-shell class - these will contain all shell commands
+var highlight_shell_elements = document.getElementsByClassName("highlight-shell");
+
+//Loop through elements with highlight-shell class and add clipboard button to each block
+for (x = 0; x < highlight_shell_elements.length; x++)
+{
+  //Create button element
+  var clipboard_button = document.createElement("BUTTON");
+
+  //Create span element and add glyphicon glyphicon-copy classes to span element
+  //Append span element as child of clipboard button
+  //Add clipboard-btn class to the clipboard button element
+  var span_element = document.createElement("SPAN");
+  span_element.setAttribute('class', 'glyphicon glyphicon-copy');
+  span_element.innerHTML = "";
+  clipboard_button.appendChild(span_element);
+  clipboard_button.setAttribute('class', 'clipboard-btn');
+
```

Comment:
Maybe we can also add an `alt-text` so that when users hover over the button (or use a screen reader) the message could say "Click to copy the command block below".